### PR TITLE
[zh-cn] update/cluster-upgrade.md

### PR DESCRIPTION
**Pull request by yanrongshi**

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@yanrongshi Please use meaningful string in the commit message.

**Comment by yanrongshi** [comment]
> @yanrongshi请在提交消息中使用有意义的字符串。

ok,thanks

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ Once you have upgraded the cluster, remember to
 -->
 如果你的集群是使用 `kubeadm` 安装工具部署而来，
 那么升级集群的详细信息，请参阅
-[升级 kubeadm 集群](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)。
+[升级 kubeadm 集群](/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:69]: 请改回

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 2]
```diff
@@ -66,10 +66,10 @@ Once you have upgraded the cluster, remember to
...
 
 升级集群之后，要记得
-[安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
+[安装最新版本的 `kubectl`](/zh/docs/tasks/tools/).
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:72]:
```suggestion
[安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/)。
```

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 3]
```diff
@@ -104,16 +104,16 @@ At this point you should
...
 kubelet on that node and bring the node back into service.
 -->
 现在，你应该
-[安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
+[安装最新版本的 `kubectl`](/zh/docs/tasks/tools/).
 
 对于集群中的每个节点，
-[排空](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)
-节点，然后，或者用一个运行了 {{< skew latestVersion >}} kubelet 的新节点替换它；
+[排空](/zh/docs/tasks/administer-cluster/safely-drain-node/)
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:115]: 请改回 zh-cn

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md (hunk 1 above)
Comment (yanrongshi) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:69]:
```suggestion
[升级 kubeadm 集群](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)。
```

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ Once you have upgraded the cluster, remember to
 [升级 kubeadm 集群](/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)。
 
 升级集群之后，要记得
-[安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
+[安装最新版本的 `kubectl`](/zh/docs/tasks/tools/).
```
Comment (yanrongshi) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:72]:
```suggestion
[安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
```

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 5]
```diff
@@ -104,16 +104,16 @@ At this point you should
...
 For each node in your cluster, [drain](/docs/tasks/administer-cluster/safely-drain-node/)
-that node and then either replace it with a new node that uses the {{< skew latestVersion >}}
-kubelet, or upgrade the {{< skew latestVersion >}}
+that node and then either replace it with a new node that uses the {{< skew currentVersion >}}
+kubelet, or upgrade the {{< skew currentVersion >}}
 kubelet on that node and bring the node back into service.
 -->
 现在，你应该
-[安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
+[安装最新版本的 `kubectl`](/zh/docs/tasks/tools/).
```
Comment (yanrongshi) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:112]:
```suggestion
[安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
```

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 6]
```diff
@@ -104,16 +104,16 @@ At this point you should
...
+kubelet, or upgrade the {{< skew currentVersion >}}
 kubelet on that node and bring the node back into service.
 -->
 现在，你应该
 [安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
 
 对于集群中的每个节点，
-[排空](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)
-节点，然后，或者用一个运行了 {{< skew latestVersion >}} kubelet 的新节点替换它；
+[排空](/zh/docs/tasks/administer-cluster/safely-drain-node/)
```
Comment (yanrongshi) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:115]:
```suggestion
[排空](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)
```

**Comment by tengqm** [comment]
/lgtm
/approve

Next time, please keep your PRs at a single commit.

**Pull request merged**